### ISSUE TITLE
Support for getting provider version info from dependency lock file

### ISSUE THREAD

**Issue by brettcurtis** [terraform-docs/terraform-docs#443]
### What problem are you facing?

Versions do not show up for providers using the new dependency lock file: https://www.terraform.io/docs/language/dependency-lock.html#dependency-installation-behavior

### How could terraform-docs help solve your problem?

Grab the version from the .terraform.lock.hcl

```none
# This file is maintained automatically by "terraform init".
# Manual edits may be lost in future updates.

provider "registry.terraform.io/hashicorp/google" {
  version = "3.61.0"
  hashes = [
    "h1:qwvjTMYJPkTvCcEJNNYBpC0RrjyzH5J3UZ4prPXX45o=",
    "zh:09ab9579b5dbb866181d33272eeb53a4aedb70f91f80921b1bc7f9b7b3409671",
    "zh:0c36fad05dad6284746025bab139bbcfd90c5caa20004f49f9ba10b23e353150",
    "zh:1e1a96f2777cd088d3018f5a1df3a420acb9cbc81ade46a454290e4ac5a18a28",
    "zh:2321d64aafde14108ba83249368076a6e756df1268a2e4638ebcbab05c565948",
    "zh:314ce3e4b04a25a438256f6b09c0e9e3e9d1e91e44366c54d914a8bfe488324d",
    "zh:56e71910f66ad71373e00981c9c568cf75e7b844d028d98c3f04a6aca87461b5",
    "zh:65543fc0ad30dd98e551a35c0c9f8d7d5de51ce69c90594f85e03edafbcd56a9",
    "zh:95cd5bec37038aec219b74ec4478eef52143895ddf75bfd796895444f0db0e87",
    "zh:e0852c06803ef9731969631da3189154a6ef9e56087bad763d3c0aa2fe99c0a4",
    "zh:f6a43e7251b3d1fad1958be7ecb99542357648da87f5f0d93f08ac5f4378736a",
  ]
}
```

**Comment by khos2ow** [comment]
That's a very good suggestion, I can see it come in handy to complement #236.

**Comment by khos2ow** [comment]
This is now added in #509 and will be available in `v0.14.0`.
